manuals/2.3/index.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: manual
+title: Foreman Nightly Manual
+version: "2.3"
+previous_version: "2.2"
```

Comment:
`manuals/nightly/index.md` should also get updated `previous_version`

Reply:
corrected the title and added previous_version